Podfile

```diff
@@ -19,15 +19,31 @@ end
 
 target 'FunctionalTests' do
   pod 'AEPCore'
-  pod 'AEPEdgeIdentity'
-  pod 'AEPEdgeConsent'
+  pod 'AEPEdgeIdentity', :git => 'https://github.com/adobe/aepsdk-edgeidentity-ios.git', :branch => 'feature/tvos'
```

Comment:
You will remove the branch path in that later PR?

Reply:
Yes, after all edge extensions are in staging, I am going to update with staging url for all the repos. We will then update this after we release all the 3 extensions